reinforcement_learning/rlclientlib/vw_model/vw_model.cc

```diff
@@ -43,19 +43,24 @@ namespace reinforcement_learning { namespace model_management {
 
       // Pick a slot using the pdf. NOTE: sample_after_normalizing() can change the pdf
       uint32_t chosen_index;
-      auto const scode = e::sample_after_normalizing(rnd_seed, std::begin(pdf), std::end(pdf), chosen_index);
+      auto scode = e::sample_after_normalizing(rnd_seed, std::begin(pdf), std::end(pdf), chosen_index);
```

Comment:
the whole code block {action_ids, pdf} -> ranking_response looks copy-pasted in vw_model and live_model_impl. Isn't it better to dedup it in some helper class / method?

Reply:
There is a subtle diff.  one case uses indexes, the other uses action_id[idx].  I found a bug here so thanks for the heads up.